pkg/formats/influx/influx.go

```diff
@@ -591,6 +593,31 @@ func (f *InfluxFormat) fromSnmpInterfaceMetric(in *kt.JCHF) []InfluxData {
 									Tags:        attrNew,
 								})
 							}
+							if !util.DropOnFilter(attrNew_bitrate, f.lastMetadata[in.DeviceName], true) {
```

Comment:
Drop on filter will also be the same across here so maybe move all these metrics up into the common if statement for attrNew?